sros2/sros2/verb/create_permission.py

```diff
@@ -31,17 +31,17 @@ class CreatePermissionVerb(VerbExtension):
     """Create permission."""
 
     def add_arguments(self, parser, cli_name):
-        arg = parser.add_argument('ROOT', help='root path of keystore')
+        arg = parser.add_argument('-k', '--keystore-root-path', help='root path of keystore')
```

Comment:
This changes a required argument to an optional one, no? I don't see that actually being handled in the code. What happens if this isn't specified?